scripts/dayenu_delay_filter_run.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+# Copyright 2019 the HERA Project
+# Licensed under the MIT License
+
+"""Command-line drive script for hera_cal.delay_filter"""
+
+from hera_cal import delay_filter
+import sys
+import pickle
+import random
+
+parser = delay_filter.delay_filter_argparser()
+a = parser.parse_args()
+
+# set kwargs
+filter_kwargs = {'standoff': a.standoff, 'horizon': a.horizon, 'tol': a.tol,
+                 'skip_wgt': a.skip_wgt, 'min_dly': a.min_dly}
+if a.window == 'tukey':
+    filter_kwargs['alpha'] = a.alpha
+
+# Run Delay Filter
+if a.partial_load_Nbls is not None:  # partial loading
+    delay_filter.partial_load_dayenu_delay_filter_and_write(a.infilename, calfile=a.calfile,
+                                                            Nbls=a.partial_load_Nbls, spw_range=a.spw_range,
+                                                            cache_dir=a.cache_dir, res_outfilename=a.res_outfilename,
+                                                            clobber=a.clobber, update_cache=a.update_cache,
+                                                            add_to_history=' '.join(sys.argv), **filter_kwargs)
+else:
+    cache_dir = a.cache_dir
```

Comment:
This is way too much code to sit outside unit testing. All of this cache handling should happen in a run function inside `delay_filter.py`

Reply:
(or possibly in io.py)

Reply:
I've moved this to io.py